2024-06-18 09:13:40,415: asyncio: INFO: [MLTrader] Executing the initialize lifecycle method
2024-06-18 09:13:41,632: root: INFO: Current backtesting datetime 2023-12-15 08:30:00-05:00
2024-06-18 09:13:41,633: asyncio: INFO: [MLTrader] Executing the before_market_opens lifecycle method
2024-06-18 09:13:41,634: root: INFO: Current backtesting datetime 2023-12-15 09:30:00-05:00
2024-06-18 09:13:41,634: asyncio: INFO: [MLTrader] Executing the before_starting_trading lifecycle method
2024-06-18 09:13:41,636: asyncio: INFO: [MLTrader] [34mExecuting the on_trading_iteration lifecycle method at 2024-06-18 09:13:41[0m
2024-06-18 09:13:41,638: root: WARNING: quote is not implemented for YahooData, but USD was passed as the quote
2024-06-18 09:13:42,537: root: WARNING: quote is not implemented for YahooData, but USD was passed as the quote
2024-06-18 09:13:42,537: asyncio: ERROR: [MLTrader] Strategy.create_order() got an unexpected keyword argument 'stop_lost_price'
2024-06-18 09:13:42,567: asyncio: ERROR: [MLTrader] Traceback (most recent call last):
  File "C:\Users\cordo\AppData\Local\Programs\Python\Python312\Lib\site-packages\lumibot\strategies\strategy_executor.py", line 923, in run
    self._run_trading_session()
  File "C:\Users\cordo\AppData\Local\Programs\Python\Python312\Lib\site-packages\lumibot\strategies\strategy_executor.py", line 871, in _run_trading_session
    self._on_trading_iteration()
  File "C:\Users\cordo\AppData\Local\Programs\Python\Python312\Lib\site-packages\lumibot\strategies\strategy_executor.py", line 262, in func_output
    result = func_input(self, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cordo\AppData\Local\Programs\Python\Python312\Lib\site-packages\lumibot\strategies\strategy_executor.py", line 286, in func_output
    result = func_input(self, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cordo\AppData\Local\Programs\Python\Python312\Lib\site-packages\lumibot\strategies\strategy_executor.py", line 397, in _on_trading_iteration
    raise e
  File "C:\Users\cordo\AppData\Local\Programs\Python\Python312\Lib\site-packages\lumibot\strategies\strategy_executor.py", line 369, in _on_trading_iteration
    on_trading_iteration()
  File "C:\Users\cordo\AppData\Local\Programs\Python\Python312\Lib\site-packages\lumibot\tools\decorators.py", line 62, in func_output
    frame, result = call_function_get_frame(func_input, *args, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cordo\AppData\Local\Programs\Python\Python312\Lib\site-packages\lumibot\tools\decorators.py", line 30, in call_function_get_frame
    result = func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\cordo\.vscode\pythonCode\tradingBot.py", line 36, in on_trading_iteration
    order = self.create_order(
            ^^^^^^^^^^^^^^^^^^
TypeError: Strategy.create_order() got an unexpected keyword argument 'stop_lost_price'

2024-06-18 09:13:42,570: asyncio: INFO: [MLTrader] Executing the on_bot_crash event method
2024-06-18 09:13:42,589: root: INFO: Backtesting finished
2024-06-18 09:13:43,191: asyncio: WARNING: [MLTrader] Cannot plot returns because the strategy returns are missing
2024-06-18 09:13:43,346: asyncio: WARNING: [MLTrader] Cannot create a tearsheet because the strategy returns are missing
2024-06-18 09:13:43,347: backtest_stats: INFO: Backtest took 0:00:02.933676 for a speed of 0.000
